Source/missiles.cpp

```diff
@@ -3622,16 +3594,16 @@ void MI_Reflect(int i)
 
 static void MissileRing(int i, int type)
 {
-	int b = CrawlNum[3];
 	missile[i]._miDelFlag = true;
-	int src = missile[i]._micaster;
-	int k = CrawlNum[3] + 1;
+	int8_t src = missile[i]._micaster;
 	uint8_t lvl = src > 0 ? plr[src]._pLevel : currlevel;
 	int dmg = 16 * (GenerateRndSum(10, 2) + lvl + 2) / 2;
 
-	for (int j = CrawlTable[b]; j > 0; j--, k += 2) {
-		int tx = missile[i]._miVar1 + CrawlTable[k - 1];
-		int ty = missile[i]._miVar2 + CrawlTable[k];
+	int k = CrawlNum[3];
+	int ck = k + 1;

```

Comment:
This seams to be off by 1 compared to all other usages of `CrawlTable`.

Reply:
Turned out to be a bug in Devilution: https://github.com/diasurgical/devilution/pull/2237